src/AddTodoForm.js

```diff
@@ -0,0 +1,16 @@
+import { React } from "react";
+
+const AddTodoForm = () => {
+    return (
+        <form>
+            <label>Title</label>
+            <input id="todoTitle" type="text" htmlFor="todoTitle"></input>
```

Comment:
Thank you Gonzalo! I moved the htmlFor to the label instead and changed the label tag itself. So HTMLfor is only used for labels?

Reply:
Well htmlFor is primarily used with labels, it's worth noting that it can technically be used with any element that has an id attribute and the corresponding element for which you want to establish a relationship. However, it's most commonly used with labels to enhance the accessibility and user experience of forms.

Reply:
> Thank you Gonzalo! I moved the htmlFor to the label instead and changed the label tag itself. So HTMLfor is only used for labels?

Well htmlFor is primarily used with labels, it's worth noting that it can technically be used with any element that has an id attribute and the corresponding element for which you want to establish a relationship. However, it's most commonly used with labels to enhance the accessibility and user experience of forms.